src/jswrap_flash.c

```diff
@@ -114,8 +116,10 @@ doesn't return all `0xFF`) you'll need to call `erasePage` to clear the
 entire page.
  */
 void jswrap_flash_write(JsVar *data, int addr) {
-  if (jsvIsUndefined(data))
-    return jsExceptionHere(JSET_ERROR, "Data is not defined");
+  if (jsvIsUndefined(data)) {
+    jsExceptionHere(JSET_ERROR, "Data is not defined");
```

Comment:
I found it confusing 'cause jsExceptionHere is not returning anything and I was changing jsExceptionHere. But you're right, I shouldn't have gratuitously modified it.